Jint/Engine.cs

```diff
@@ -42,6 +42,30 @@ public class Engine
 
         // cache of types used when resolving CLR type names
         internal Dictionary<string, Type> TypeCache = new Dictionary<string, Type>();
+		// cache of learned type conversions.
```

Comment:
Please fix the indentation. There are lots of occurrences like this where you replaced spaces with tabs.